connectors/src/start_worker.ts

```diff
@@ -4,6 +4,7 @@ import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
 
 import { runConfluenceWorker } from "@connectors/connectors/confluence/temporal/worker";
+import { runMicrosoftWorker } from "@connectors/connectors/microsoft/temporal/worker";
```

Comment:
We likely don't want to start the worker yet in that stage do we?

Reply:
that's clearly not needed, but `runMicrosoftWorker` is just an empty placeholder for now